main.go

```diff
@@ -61,6 +61,24 @@ var (
 	envMaxNumAPIKeysPerUser = "ACCOUNTS_MAX_NUM_API_KEYS_PER_USER"
 )
 
+type (
+	// ServiceConfig represents all configuration values we expect to receive
+	// via environment variables or config files.
+	ServiceConfig struct {
+		DBCreds               database.DBCredentials
```

Comment:
f/u docstrings for fields

Reply:
https://github.com/SkynetLabs/skynet-accounts/issues/138